report-viewer/src/components/CodePanel.vue

```diff
@@ -101,44 +126,48 @@ export default defineComponent({
      * Key is line number, value is id of linked line.
      * @type {Ref<UnwrapRef<{}>>}
      */
-    const linksArray = ref({})
+    const linksArray = ref({});
     /**
      * Indicates whether the line is last line of match. Key is line number, value is true or false.
      * @type {Ref<UnwrapRef<{}>>}
      */
-    const isLast = ref({})
+    const isLast = ref({});
     /**
      * Indicates whether the line is the first line of a match. Key is line number, value is true or false.
      * @type {Ref<UnwrapRef<{}>>}
      */
-    const isFirst = ref({})
+    const isFirst = ref({});
 
     /**
      * Initializing the the upper arrays.
      */
-    props.matches.forEach(m => {
+    props.matches.forEach((m) => {
       for (let i = m.start; i <= m.end; i++) {
         //assign match color to line
-        coloringArray.value[i] = m.color
+        coloringArray.value[i] = m.color;
         //assign link object to line.
-        linksArray.value[i] = {panel: m.linked_panel, file: m.linked_file, line: m.linked_line}
+        linksArray.value[i] = {
+          panel: m.linked_panel,
+          file: m.linked_file,
+          line: m.linked_line,
+        };
         //check whether line is start of match and assign true if yes
         if (i === m.start) {
-          isFirst.value[i] = true
+          isFirst.value[i] = true;
         }
         //check whether line is end of match and assign true if yes
         if (i === m.end) {
-          isLast.value[i] = true
+          isLast.value[i] = true;
         }
       }
```

Comment:
I know this is not your code, but I suggest taking this part out of the loop and using `m.start` and `m.end` as indices directly.
```suggestion
      }
      isFirst.value[m.start] = true;
      isLast.value[m.end] = true;
```